### PR TITLE
Updated to match final implementation.

### PR DESCRIPTION
**Pull request by alanconway**
See https://issues.redhat.com/browse/LOG-3982

/cc jewzaam

### CODE REVIEW

File: enhancements/cluster-logging/api-audit-log-policy.md [hunk 1]
```diff
@@ -89,57 +91,40 @@ See [Kubernetes Documentation][k8s-auditing] for an example.
...
-
-This problem is not addressed by this proposal, but needs to be addressed.
-
-Some avenues to explore:
-- Create separate operators using different API groups to distinguish the API types. \
-  Similar approach was used for the observability opertor:
-  - [MON-2792 OBO support for monitoring.coreos.com group](https://issues.redhat.com/browse/MON-2792)
-  - [HOSTEDCP-624 Support monitoring.rhobs Resources](https://issues.redhat.com/browse/HOSTEDCP-624)
-- OLM 1.x has multi-operator-version features
+None.
```
Comment (jewzaam) [inline enhancements/cluster-logging/api-audit-log-policy.md:127]: How did the risk change here to "none"?

File: enhancements/cluster-logging/api-audit-log-policy.md [hunk 2]
```diff
@@ -171,30 +155,35 @@ Both can scrape from the same audit log files simultaneously, and perform separa
...
-- New dependency to be maintained for the log collector.
+- Increased scope for user error in policies.
+- Possible additional CPU cost.
 
 ## Design Details
 
-### Open Questions
-
-#### Form of audit policy
+None.
```
Comment (jewzaam) [inline enhancements/cluster-logging/api-audit-log-policy.md:163]: No design details or should this be an empty higher level section?

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "In Progress". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "In Progress". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "In Progress". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "In Progress". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "In Progress". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "In Progress". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "In Progress". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Comment by dhellmann** [comment]
(automated message) This pull request is closed with lifecycle/rotten. The associated Jira ticket, OBSDA-344, has status "Closed, Done". Should the PR be reopened, updated, and merged? If not, removing the lifecycle/rotten label will tell this bot to ignore it in the future.

**Pull request closed without merging**